internal/app/api/v1/server.go

```diff
@@ -307,9 +308,21 @@ func (s *TestkubeAPI) InitRoutes() {
 	configs.Get("/", s.GetConfigsHandler())
 	configs.Patch("/", s.UpdateConfigsHandler())
 
+	debug := s.Routes.Group("/debug")
+	debug.Get("/listeners", s.GetDebugListenersHandler())
```

Comment:
Is this for final release?

Reply:
yeah let's keep it in case of some issues, we can remove it later